shell/pages/prefs.vue

```diff
@@ -153,6 +155,13 @@ export default {
     <div class="mt-10">
       <t k="prefs.theme.autoDetail" :pm="pm" :am="am" />
     </div>
+    <hr />
+    <h4 v-t="'prefs.language'" class="mt-10" />
+    <div class="row">
+      <div class="col span-4">
+        <LocaleSelector></LocaleSelector>
```

Comment:
If there's no content for an element it's usually better to close it within the first tag `<LocaleSelector/>` (same goes for shell/pages/auth/login.vue)

Reply:
Fixed